plugins/woocommerce/includes/admin/views/html-notice-legacy-shipping.php

```diff
@@ -22,7 +22,7 @@
 	</p>
 
 	<p class="submit">
-		<?php if ( empty( $_GET['page'] ) || empty( $_GET['tab'] ) || 'wc-settings' !== $_GET['page'] || 'shipping' !== $_GET['tab'] ) : ?>
+		<?php if ( ! is_wc_admin_settings_page() || empty( $_GET['tab'] ) || 'shipping' !== $_GET['tab'] ) : // phpcs:ignore WordPress.Security.NonceVerification.Recommended ?>
```

Comment:
Note: same as original, but adds `is_admin()` check, which looks safe based on the usage context.